cou/utils/juju_utils.py

```diff
@@ -46,6 +46,9 @@
 )
 from cou.utils.openstack import is_charm_supported
 
+# Default for the Juju MAX_FRAME_SIZE to be 256MB to stop
+# "RPC: Connection closed, reconnecting" errors and then a failure in the log.
+# See https://github.com/juju/python-libjuju/issues/458 for more details
 JUJU_MAX_FRAME_SIZE: int = 2**30
```

Comment:
What is the unit of `JUJU_MAX_FRAME_SIZE` and how is it related to the comment you add back?

Reply:
Free feel to resolve it when you have or don't have the answer 🙂 but we should keep the comment correct

Reply:
If the source code is correct: https://github.com/juju/python-libjuju/blob/master/juju/client/connection.py#L231-L242, then the unit should be `byte`